### PR TITLE
Adjusting parameters to the new pugin.

### PR DESCRIPTION
**Pull request by dmaluenda**
This PR contains all the changes I made in order to accommodate the monitors to its own plugin.

I've checked the tests and they pass.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,4 +1,6 @@
 scipion-pyworkflow
```
Comment (pconesa) [inline requirements.txt:1]: I'd leave pyworkflpw out since scipion-em should have it.

File: README.rst [hunk 2]
```diff
@@ -1,45 +1,51 @@
...
-See https://scipion-em.github.io/docs/release-3.0.0/docs/facilities/customize-report.html for more information
 
--------------
-Development
--------------
+Installation
+------------
 
-To install **scipion4facilities** for development purposes, one can do:
+You will need to use `Scipion 3.0 <https://github.com/I2PC/scipion/releases/tag/V3.0.0>`_
```
Comment (pconesa) [inline README.rst:21]: Thus should point somewhere else: https://scipion-em.github.io/docs/release-3.0.0/index.html ?

**Comment by delarosatrevin** [comment]
I agree with @pconesa that emfacilities is a more consistent name for the package. The overall plugin scipion-em-facilities is fine and in line with other plugin names.

File: README.rst (hunk 2 above)
Comment (dmaluenda) [reply README.rst:21]: Right, I've copied from scipion-em. We should change it also there.

**Comment by rmarabini** [comment]
Hi,

 Secrets.py should NOT be in github.  I delete it.

**Comment by rmarabini** [comment]
Hi,

   The code is OK with me. I did a few modifications in the pull request

1) remove secrets.py which should be secrets
2) add a few lines of code  to repoert_grafana.py that I include in the monitor after David forked it
3) fix a few typos in the comments

   I think this is ready to go @dmaluenda.

**Comment by dmaluenda** [comment]
Ok, @rmarabini . I'm not totally confortable with the secrets. As it is, the whole plugin cannot be load since a failing import is found...

CHo do you see the following approach? putting secrets in a json, importing them in a dictionary, getting them with the 'get()' function by using default values if not found. In this way, the plugin works and when no secrets implemented we can raise a warning with some instructions.

Another option is putting the import from secrets in a try/catch.

**Comment by pconesa** [comment]
Yes, we should be careful with importing things that might fail.

**Comment by rmarabini** [comment]
Hi,


  I am not happy with secrets.py either. We certainly need something  more
configurable. Anyway my suggestion is, first let us finish with this pull
request and them we modify the way secrets.py is uploded. Certainly, it
should be a config  file and we are going to need a environment variable
that  allow us to place it in places different from the default one.

   Roberto

**Pull request merged**